zio/csvio/writer.go

```diff
@@ -0,0 +1,84 @@
+package csvio
+
+import (
+	"encoding/csv"
+	"errors"
+	"io"
+	"time"
+
+	"github.com/brimsec/zq/pkg/nano"
+	"github.com/brimsec/zq/zng"
+	"github.com/brimsec/zq/zng/flattener"
+	"github.com/brimsec/zq/zng/resolver"
+)
+
+var ErrNotDataFrame = errors.New("Cannot format non-dataframe group of records as csv.")
```

Comment:
```suggestion
var ErrNotDataFrame = errors.New("different record types seen during csv write")
```

Reply:
Ha!  My first error message was like this one but I changed it to be more data-sciency.  Will change it back.